docs/static/reference/php/renderer.index

```diff
@@ -0,0 +1 @@
s:40:"e90e3650c114a7744810e6f344149828a3a54967";s:25:"Dagger\InterfaceTypeDefId";s:40:"389f4ba6599668a38aa6a474b4fa2293f9fb3fe3";s:11:"Dagger\Json";s:40:"6e52f55cbaec3227fb566937a3125e1e4372342e";s:10:"Dagger\LLM";s:40:"91debfa95d1a2d7302c86d66f5c77011d933a7a3";s:12:"Dagger\LLMId";s:40:"8276d5be767664cddf85d8b2b380e43ecc975d5c";s:20:"Dagger\LLMTokenUsage";s:40:"f05faeaa2d4f0e2aa3bedd7a92404a954c48cddf";s:22:"Dagger\LLMTokenUsageId";s:40:"9a888ea6e08c4facc8c9bcf3a8b90f919f3ab575";s:12:"Dagger\Label";s:40:"7d08dc0f0cce1fc0a015263e740721d80eba259b";s:14:"Dagger\LabelId";s:40:"3dc472658ff734743971535ad954dc7903fd0469";s:18:"Dagger\ListTypeDef";s:40:"a045a67134f2930bc4097e389c4d50b3db62e15a";s:20:"Dagger\ListTypeDefId";s:40:"61ad34c3ed73445042e6f40fa93434c3825c0d02";s:13:"Dagger\Module";s:40:"ce8762d3a0279d2ebb4ec064aa9616e6f1950953";s:25:"Dagger\ModuleConfigClient";s:40:"141d0e1267ec444b5d4d10abea48822250dda423";s:27:"Dagger\ModuleConfigClientId";s:40:"bfa70ca8951238848ec381b58277cd3686f88c64";s:15:"Dagger\ModuleId";s:40:"eae2161e7344840a16f0a115c587ecd84729f779";s:19:"Dagger\ModuleSource";s:40:"500c8e185c770d3e0b2bc3ccc0fe5cd82781af24";s:21:"Dagger\ModuleSourceId";s:40:"e42b13465b1dc6019b5657c07e9ec4d0b69d8102";s:20:"Dagger\ObjectTypeDef";s:40:"c8e8f590236bd7c7a4fabf949e30fe032204a478";s:22:"Dagger\ObjectTypeDefId";s:40:"68efd4f65fb4320692c555c65bf0cae88ae59d43";s:20:"Dagger\PipelineLabel";s:40:"7ecd4805b71e92e9436cb2c77152ca362347d953";s:15:"Dagger\Platform";s:40:"bf145c3bbb4d105a1ed182233a525dc74448bb39";s:11:"Dagger\Port";s:40:"230147d5022892945405c47e001226201ac6f329";s:18:"Dagger\PortForward";s:40:"d043b91a6aeddc222f98dc3da99d0192aed00f69";s:13:"Dagger\PortId";s:40:"4337a4444768f04d0a8c4ea5bd9fa46bae6019ef";s:16:"Dagger\SDKConfig";s:40:"9bbb25c5525248dcb21b2aac7e0211639c13978e";s:18:"Dagger\SDKConfigId";s:40:"d821c0833382a7346b5a9d9d466400be8b50ff53";s:20:"Dagger\ScalarTypeDef";s:40:"d6d1e31d44b46cf458482edef303987a54859c14";s:22:"Dagger\ScalarTypeDefId";s:40:"2ede70846fda4c2424b8662f25c0d0fa92d0af4a";s:13:"Dagger\Secret";s:40:"bd4c10576c8efc5a8fbf7a45b4dc2f701db53167";s:15:"Dagger\SecretId";s:40:"d1ab3e44242471724daeedb9f40d2af3919f7aaa";s:14:"Dagger\Service";s:40:"d1cac79daee3143260b1da2fcfcf2a72db271da6";s:16:"Dagger\ServiceId";s:40:"ef33c82eb8d5f95b6007db7bc5dcbec84c4b5103";s:13:"Dagger\Socket";s:40:"c6f797d5bfd84890fe67b0b7377b790026edac25";s:15:"Dagger\SocketId";s:40:"63a510e5c175aa5bc88c5efb9de9acc20c6203fa";s:16:"Dagger\SourceMap";s:40:"97ac4c1c3d2472a5352b6b3fcc3ba382e448bff0";s:18:"Dagger\SourceMapId";s:40:"316469285d33887e2c708dedf4f5542dba2d7614";s:15:"Dagger\Terminal";s:40:"130ea9d9953fff5c7740b70f46aaf91e367ad3cd";s:17:"Dagger\TerminalId";s:40:"b742dd1fbd01daea8c49f17be3458e05f9d9b404";s:14:"Dagger\TypeDef";s:40:"9e98ca7a53a88dc6eec1ed8bdf8878d2e3a1d415";s:16:"Dagger\TypeDefId";s:40:"fefe81a49d75e7ba6376ec76563143cd999ca8c1";}i:1;a:1:{i:0;s:4:"main";}i:2;a:3:{i:0;s:6:"Dagger";i:1;s:16:"Dagger\Attribute";i:2;s:13:"Dagger\Client";}}
```

Comment:
@grouville check complained about this file being present (`Only in a/docs/static/reference/php: renderer.index`). Should it be removed?